content/updates/2020-11/4-Progress-bar-and-time-left-for-your-ongoing-jobs.md

```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_alt_img: "The new progress bar in the process tracker"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Progress bar and time left for your ongoing jobs
+::: meta-data type="New" features="Productivity" available="mid-November" link-to-doc="../articles/TODO.html"
+:::
+
+You are now able to see an estimation of the remaining time and a progress bar for all your ongoing jobs. So don't hesitate to have a look at your process tracker!
```

Comment:
```suggestion
Whenever you launch a job, a progress bar and an estimation of the remaining time before the job is completed, are now displayed in the process tracker. You can now easily see how long it will take to process a job, to help you better organize your daily tasks. 
```
I should probably wait until Thursday to check the feature live. I don't know where the progress bar, time left will be visible.(process tracker page or job execution detail page?). Also, a user can see the progress bar and time left for ALL jobs or only his/her jobs? cc @felix-couprie 
Félix, you will also have to redo the screenshots that show the process tracker page the 'started', 'completed' and 'failed' - since badges have changed - on this page: https://help.akeneo.com/pim/serenity/articles/monitor-jobs.html#how-to-monitor

Reply:
It's in the job execution detail page. So only users who have access to the job can see the progress.